### ISSUE TITLE
Element com.mikepenz.iconics.IconicsDrawable must be declared

### ISSUE THREAD

**Issue by INGUANA** [mikepenz/Android-Iconics#493]
I can't render the image in a IconicsImageView component. My application is running but the icon won't appear. For example, i try to implement the example with the gmd-favorite icon. I declare dependencies in gradle, make a drawable like the example, make an IconicsImageView component like the example, run it and it doesnt show up.

drawable icon layout:
`<?xml version="1.0" encoding="utf-8"?>
<com.mikepenz.iconics.IconicsDrawable
    xmlns:app="http://schemas.android.com/apk/res-auto"
    app:ico_icon="gmd_favorite"
    app:ico_color="#0000FF"
    app:ico_contour_width="1dp"
    app:ico_contour_color="#00FFFF"
    app:ico_size="24dp" />`

IconicsImageView component:
`<com.mikepenz.iconics.view.IconicsImageView
        android:layout_width="72dp"
        android:layout_height="72dp"
        android:layout_marginEnd="@dimen/margin_end"
        android:layout_marginBottom="@dimen/margin_bottom"
        app:iiv_color="@android:color/holo_red_dark"
        app:iiv_icon="gmd-favorite"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintEnd_toEndOf="parent"/>`

Note that inside the drawable xml, the component nametag is marked as red and says "Element com.mikepenz.iconics.IconicsDrawable must be declared". I've created the same xml for API lvl 24 aswell since this approach was implemented for API 24+ but no luck.

I'm not sure the root of the cause, so i downloaded this project from git to check for differences and (besides the fact that it is written in kotlin, while i use java) it had the same red flagging on the drawable of gmd-favorite. Are there any suggestions on what the issue might be? Below I'm posting the module gradle file.

USING:
- Android Studio, Gradle Plugin 3.5.3
- Java 1.8
- Gradle 5.4.1

````
apply plugin: 'com.android.application'
apply plugin: 'com.google.gms.google-services'

android {
    compileSdkVersion 28
    defaultConfig {
        applicationId "arandomid"
        minSdkVersion 24
        targetSdkVersion 28
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
        }
    }
    compileOptions {
        sourceCompatibility = 1.8
        targetCompatibility = 1.8
    }
}

dependencies {

    .  .  .
    implementation 'androidx.appcompat:appcompat:1.1.0'
    implementation "com.mikepenz:iconics-core:3.2.5"
    implementation "com.mikepenz:iconics-views:3.2.5"

    implementation 'com.mikepenz:google-material-typeface:3.0.1.4.original-kotlin@aar'
    implementation 'com.mikepenz:material-design-iconic-typeface:2.2.0.6-kotlin@aar'
    implementation 'com.mikepenz:fontawesome-typeface:5.9.0.0-kotlin@aar'
    implementation 'com.mikepenz:octicons-typeface:3.2.0.6-kotlin@aar'
    implementation 'com.mikepenz:meteocons-typeface:1.1.0.5-kotlin@aar'
// note 3.7.95.4 alphabetically sorts, and merges in only 2 sections, split between G and H
    implementation 'com.mikepenz:community-material-typeface:3.7.95.4-kotlin@aar'
    implementation 'com.mikepenz:weather-icons-typeface:2.0.10.5-kotlin@aar'
    implementation 'com.mikepenz:typeicons-typeface:2.0.7.5-kotlin@aar'
    implementation 'com.mikepenz:entypo-typeface:1.0.0.5-kotlin@aar'
    implementation 'com.mikepenz:devicon-typeface:2.0.0.5-kotlin@aar'
    implementation 'com.mikepenz:foundation-icons-typeface:3.0.0.5-kotlin@aar'
    implementation 'com.mikepenz:ionicons-typeface:2.0.1.5-kotlin@aar'
    implementation 'com.mikepenz:pixeden-7-stroke-typeface:1.2.0.3-kotlin@aar'
    implementation 'com.mikepenz:material-design-icons-dx-typeface:5.0.1.0-kotlin@aar'

}
```
`

**Comment by mikepenz** [comment]
@INGUANA sadly this feature does not exist in v3.2.5 it was only introduced in the latest release. 

When you use a library with a specific version please always use the the README, docs and sample with the according version selected.

![image](https://user-images.githubusercontent.com/1476232/75092812-dcaec100-557b-11ea-9455-c1a212d64a89.png)

This new feature is only available starting with v5.0.0

Thank you so much

**Comment by INGUANA** [comment]
Oh i see! Thank you for the response and pointing this out, i'll be taking this approach from now on!
